code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1378,14 +1378,14 @@ GLOBAL_LIST_EMPTY(roundstart_race_names)
 
 	if(gravity && !flight)	//Check for chemicals and innate speedups and slowdowns if we're on the ground
 		if(HAS_TRAIT(H, TRAIT_GOTTAGOFAST))
-			. -= 1
+			. -= 0.5
 		if(HAS_TRAIT(H, TRAIT_GOTTAGOREALLYFAST))
-			. -= 2
+			. -= 1
 		. += speedmod
 		. += H.physiology.speed_mod
 
 	if (H.m_intent == MOVE_INTENT_WALK && HAS_TRAIT(H, TRAIT_SPEEDY_STEP))
-		. -= 1.5
+		. -= 0.75
```

Comment:
Why nerf this tho?

Reply:
Sorry I didn't respond faster, but I was essentially given a go ahead to nerf **all** move speed buffs that aren't sprinting.
That being said, I'm willing to back down on this one, provided it's a rarely taken quirk that stops working when sprinting. In hindsight, it's not a contributing factor to the problem Kevinz and I want to address.